Content.Server/Explosion/Components/OnTrigger/AnchorOnTriggerComponent.cs

```diff
@@ -0,0 +1,13 @@
+using Content.Server.Explosion.EntitySystems;
+
+namespace Content.Server.Explosion.Components;
+
+/// <summary>
+/// Will anchor the attached entity upon a <see cref="TriggerEvent"/>.
+/// </summary>
+[RegisterComponent]
+public sealed class AnchorOnTriggerComponent : Component
+{
+    [DataField("removeOnTrigger")]
+    public bool RemoveOnTrigger = true;
```

Comment:
Remove this.

Reply:
Why? It is required to not trigger it at the end of second stage